tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    !pip install --upgrade --force-reinstall grpcio==1.56.0
</pre><p>Is this update of the grpcio dependency still required, or was that only needed for Milvus lite?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>this is still required</p>